Drop-In/src/androidTest/java/com/braintreepayments/api/CardDetailsFragmentUITest.kt

```diff
@@ -37,17 +37,47 @@ class CardDetailsFragmentUITest {
     }
 
     @Test
-    fun whenStateIsRESUMED_expirationDateFieldIsFocused() {
+    fun whenStateIsRESUMED_andCardholderNameDisabled_expirationDateFieldIsFocused() {
+        val dropInRequest = DropInRequest()
         val args = Bundle()
-        args.putParcelable("EXTRA_DROP_IN_REQUEST", DropInRequest())
+        args.putParcelable("EXTRA_DROP_IN_REQUEST", dropInRequest)
         args.putParcelable("EXTRA_CARD_FORM_CONFIGURATION", CardFormConfiguration(false, false))
         args.putString("EXTRA_CARD_NUMBER", VISA)
         val scenario = FragmentScenario.launchInContainer(CardDetailsFragment::class.java, args, R.style.bt_drop_in_activity_theme)
         scenario.moveToState(Lifecycle.State.RESUMED)
 
+        assertEquals(CardForm.FIELD_DISABLED, dropInRequest.cardholderNameStatus)
```

Comment:
nit: It feels like the `DropInRequest` constructor should be responsible for setting the default `FIELD_DISABLED` value.

Reply:
Yeah the constructor does set it to FIELD_DISABLED by default, just wanted to make that explicit in the test, but I can remove the assertion since it is the default 👍